streams-bootstrap-core/src/main/java/com/bakdata/kafka/ProducerApp.java

```diff
@@ -46,4 +45,7 @@ default ProducerCleanUpConfiguration setupCleanUp(
             final EffectiveAppConfiguration<ProducerTopicConfig> configuration) {
         return new ProducerCleanUpConfiguration();
     }
+
+    @Override
+    SerializerConfig defaultSerializationConfig();
```

Comment:
Can you add a javadoc? Also for StreamsApp.

Reply:
It is inherited from [App](https://github.com/bakdata/streams-bootstrap/pull/239/files#diff-08a4a32d60f8228190e670de077f3b54219d31c25fb8b4510e063f432c9a35cfR70)

Reply:
I just enforce return type here to be SerializerConfig and not the interface